website/src/components/about/AboutPage.js

```diff
@@ -36,7 +36,7 @@ function AboutPage() {
 
   const faq = {
     "Is this official?": "No, this is not official. However, the data is from official sources. Look below for more info.",
-    "Who are you?": "My name is Shreyas, and I am a student currently in the 11th grade and studying at Delhi Public School Bangalore East in Bangalore.",
+    "Who made this?": "My name is Shreyas, and I am a student currently in the 11th grade and studying at Delhi Public School Bangalore East in Bangalore. I started this project, but it has since become open source, meaning other developers from around the world also contribute to this website.",
```

Comment:
Do you feel it should be "but it has since become open source" or "but it has since been made open source"?